thirtyinch/proguard-rules.txt

```diff
@@ -1,2 +1,3 @@
 # ThirtyInch
--keep public class * implements net.grandcentrix.thirtyinch.distinctuntilchanged.DistinctComparator
\ No newline at end of file
+-keep public class * implements net.grandcentrix.thirtyinch.distinctuntilchanged.DistinctComparator
+-keep class androidx.fragment.app.Fragment { *; }
```

Comment:
Why do we have do keep it now? 🤔

Reply:
Good point, we don't need it anymore. We needed that in AndroidX projects while Ti isn't AndroidX ready.